code/game/objects/items/devices/radio/radio.dm

```diff
@@ -7,6 +7,7 @@
 	var/on = 1 // 0 for off
 	var/last_transmission
 	var/frequency = 1459 //common chat
+	var/new_frequency
```

Comment:
This is only used inside `Topic`, right? Why is this a datum var instead of being declared inside `Topic`?